runtime/include/j9cfg.h.ftl

```diff
@@ -43,6 +43,21 @@ extern "C" {
 
 #define JAVA_SPEC_VERSION ${uma.spec.properties.JAVA_SPEC_VERSION.value}
 
+/* Ensure J9VM_JAVA9_BUILD is not defined elsewhere. */
```

Comment:
Is similar code required in j9cfg.h.in for cmake?

Reply:
In the not-to-distant future, I hope to remove `J9VM_JAVA9_BUILD` entirely.